projects/storefrontapp-e2e-cypress/cypress/e2e/regression/user_access/navigation-login-a11y.e2e.cy.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as login from '../../../helpers/login';
+import { viewportContext } from '../../../helpers/viewport-context';
+
+describe('Navigation Login', () => {
+  let user;
+  viewportContext(['desktop'], () => {
+    before(() => {
+      cy.visit('/login');
+      user = login.registerUserFromLoginPage();
+    });
+
+    it('should login and logout successfully and have correct Navigation Menu buttons values', () => {
+      login.loginUser();
+
+      const tokenRevocationRequestAlias =
+        login.listenForTokenRevocationRequest();
+
+      cy.get(
+        'cx-login > cx-page-slot > cx-navigation > cx-navigation-ui > nav > ul > li > button'
+      )
+        .contains('My Account')
```

Comment:
Maybe we can shorten these selectors and use an aliases to avoid repetitions?
e.g.:

```suggestion
      cy.get('cx-login button').contains('My Account').as('MyAccountBtn')
     // and later cy.get('@MyAccountBtn')
```

Reply:
@Pio-Bar  Addressed it and improved the test further.